### ISSUE TITLE
Tests scripts for PSB & MongoDSB on minikube not working

### ISSUE THREAD

**Issue by mrmccarthy** [ocopea/kubernetes#6]
The scripts are searching for NodePort in the output of kubectl describe svc for 

On minikube version 0.21 and 022, neither report a NodePort field.

```
$ kubectl describe svc k8spsb --namespace=testing 
Name:              k8spsb
Namespace:         testing
Labels:            <none>
Annotations:       <none>
Selector:          app=k8spsb
Type:              ClusterIP
IP:                10.0.0.197
Port:              service-http  80/TCP
TargetPort:        8080/TCP
Endpoints:         172.17.0.16:8080
Session Affinity:  None
Events:            <none>
$ kubectl describe svc mongo-k8s-dsb --namespace=testing 
Name:              mongo-k8s-dsb
Namespace:         testing
Labels:            <none>
Annotations:       <none>
Selector:          app=mongo-k8s-dsb
Type:              ClusterIP
IP:                10.0.0.144
Port:              service-http  80/TCP
TargetPort:        8080/TCP
Endpoints:         172.17.0.17:8080
Session Affinity:  None
Events:            <none>

```

**Comment by shpandrak** [comment]
@mrmccarthy How did you run the psb/dsb. if it is with the standard deploy-site it won't work. You should run those with the deploy-k8spsb/deploy-mongodsb commands.
When running with the 'deploy-site' command the psb/dsb are running as internal cluster services and accessed via ClusterIp - meaning there is no NodePort exposed.
When running those with the dploy-k8spsb/deploy-mongodsb they are exposed publically...
As the script suggest run those with the -namespace=testing

**Comment by mrmccarthy** [comment]
To clarify, I was attempting to run testMongoDsbMinikube.sh and testK8sPsbMinikube.sh against a minikube deployment to a namespace called testing.

**Comment by shpandrak** [comment]
Yes, this is ok, but the issue is that to do that you need to deploy those individually using the deployer commands 'deploy-mongodsb' and 'deploy-k8spsb' and not the 'deploy-site' which I think you are using...

**Comment by mrmccarthy** [comment]
Agreed - we should enhance the documentation to show examples of the minikube deployment type needed to run each test (i.e. not a deploy-site when testing the dsb/psb).
